postfix/assets/configuration/spec.yaml

```diff
@@ -36,8 +36,10 @@ files:
   - template: instances
     options:
     - name: directory
-      required: true
-      description: Path to the postfix directory.
+      required: false
```

Comment:
You can remove this line. If it's omitted, it defaults to false. 
```suggestion
```

Reply:
Should be addressed now. Thanks!